### DISCUSSION TITLE
New Resources page

### DISCUSSION THREAD

**Discussion by GomezIvann** [dxc-technology/halstack-react#1639] [Ideas]
Since the addition of the UXPin's classic design library and the prototyping low-code tool, based on the UXPin merge, a need came to my mind. We have a lot of assets/resources/tools related to the Halstack Design System that are spread out into multiple places. These are not always easy to find and our links to them are often hidden in our documentation. I think it would make sense to add a new page called **Resources** with quick accesses to the different tools and features we currently offer outside [Halstack](https://developer.dxc.com/halstack/next/overview/introduction/).

I link a few examples from other design systems for reference:

- https://atlassian.design/resources
- https://developer.microsoft.com/en-us/fluentui#/resources
- https://design.wonderflow.ai/get-started/resources
- https://cloudscape.design/get-started/resources/design-resources/

**Comment by kukoc** [comment]
I like the idea very much. Some of the links shared in the original message are not fully aligned with what I have in mind. Maybe Fluent is the closer one. I see room for: 

- Design assets
- Prototyping resources
- Training material? recordings?
- Contribution model/process
- Decision tree?

**Reply by GomezIvann** [reply]
I partially agree with you Aitor. My take here is to have there all the resources that are external to the Halstack site and sometimes can be hard to find. That's the first criterion, the second is to have resources that can be downloaded like fonts, images, etc.

1. Design resources and assets.
2. Prototyping resources.
3. Theme generator?
4. External page with the training materials or recordings.

Tutorials, FAQs, contribution model and the decision tree are key parts of Halstack which I think require their specific Learning section/path/page. 

I like a lot how Atlassian does it: https://atlassian.design/resources/.
